Python/Ask_How_To_Qus_using_Voice/README.md

```diff
@@ -0,0 +1,87 @@
+# Ask 'How to' Questions using Voice
+
+For asking how to type questions firstly we are using the `SpeechRecognition` library so that our device can recognize our voice command to take input and then `pyttsx3` library for converting text to speech so that the device can speak up the results or output, and last important library is `pywikihow` to fetch the answers for how to type questions. 
+
+# Packages Used
```

Comment:
Don't use multiple H1 headings, make them H2.

APplicable for all the subheadings apart from the title.

Reply:
done